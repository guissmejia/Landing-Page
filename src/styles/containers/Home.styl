.hero
  width 100%
  height auto
  padding 6% 0 0
  text-align center
  display grid
  grid-template-columns 50% 50%
  &__text 
    display grid
    grid-template-rows 30% 20% 15%
    grid-gap 10px
  &__text
    h1 
      color #BEBEBE
      align-self end
      margin-bottom 0
  &__text 
    h3 
      color #BEBEBE
  &__text 
    span 
      color #DA4A74
  &__image 
    img 
      width 90%
      height auto
  &--button 
    button 
      width 30%
      margin-top 10px
      padding 15px 10px
      background #DA4A74
      color #ffffff
      font-size 20px
      font-weight 800
      border none
      border-radius 5px
      transition all 0.3s
  &--button 
    button
      &:hover 
        background #8a072d
        transform scale(0.9)
        cursor pointer
