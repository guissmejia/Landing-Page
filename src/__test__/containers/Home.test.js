import React from 'react';
import { create } from 'react-test-renderer';
import Home from '../../containers/Home';

describe('Home Snapshot', () => {
  test('Comprobar la UI del container Home', () => {
    const home = create(<Home />).toJSON();
    expect(home).toMatchSnapshot();
  });
});
