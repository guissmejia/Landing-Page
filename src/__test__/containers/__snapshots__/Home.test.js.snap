// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot Comprobar la UI del container Home 1`] = `
<section
  className="hero"
>
  <div
    className="hero__text"
  >
    <h1>
      Hello
      <span>
        World
      </span>
    </h1>
    <h3>
      Lorem ipsum dolor sit amet, actetur adipiscing elit.
      <br />
      Aenean
      <span>
        dolor
      </span>
      nisi.
    </h3>
    <div
      className="hero--button"
    >
      <button
        type="button"
      >
        Button
      </button>
    </div>
  </div>
  <div
    className="hero__image"
  >
    <img
      alt="figure"
      src=""
    />
  </div>
</section>
`;
